pkgs/by-name/rp/rpcemu/package.nix

```diff
@@ -59,9 +59,7 @@ stdenv.mkDerivation (finalAttrs: {
       using them with RPCEmu.
     '';
     license = lib.licenses.gpl2Plus;
-    maintainers = builtins.attrValues {
-      inherit (lib.maintainers) ;
-    };
+    maintainers = with lib.maintainers; [ ];
```

Comment:
This was introduced in https://github.com/NixOS/nixpkgs/commit/cab2a1296e683e21f6aa43e048eb9d6f8b89a408#diff-e34b09f53caa0dcf521a30958db3b6f8bd06d3f174558bd4e3a5247ab0f13acf.